content/ja/handbook/eba/ceo-scheduling.md

```
@@ -0,0 +1,157 @@
+> - [価格戦略](/handbook/company/pricing/)
+> - [歴史](/handbook/company/history/)
+> - [会社について](/handbook/company/)
+> - [All Remote](/handbook/company/culture/all-remote/)
+
+1. 肯定的な回答を受け取った場合、50 分の YouTube パブリックライブストリームをスケジュールします。インタビューの例については、[リモートワークのストレスについてのこちら](https://www.youtube.com/watch?v=23XIx6n9SsQ) を参照してください。
+1. 48 時間以内に、URL を知っている人なら誰でも提案または編集権限を持つドラフト記事を Google Doc で私たちと共有してください。
+1. 公開したくないものは何でも編集できます。
+1. 私たちのエグゼクティブ管理者は、それが私たちの現在のマーケティングプランに適合し、オーディエンスにとって良いフィットであると同意した場合、記事を公開するためにあなたと協力します。EBA チームは、ドラフト記事が送信されることを確認し、ブログでクロスポストするために Content Marketing と調整します。
+1. [過去の Pick Your Brain インタビュー](/handbook/hiring/) を読み視聴してください。これには、[Hacker News のトップスポット](https://news.ycombinator.com/item?id=12615723) に達した [Slab 共同創業者の Jason Chen がホストしたもの](https://about.gitlab.com/blog/2016/07/14/building-an-open-source-company-interview-with-gitlabs-ceo/) が含まれています。
+
+Email への返信: チャットしたいというお気持ちありがとうございます。私たちの会話から他の人々も恩恵を受けられるよう、GitLab の [ハンドブック](#pick-your-brain-meetings) で提案されている形式でミーティングを行うことを提案します。それでよろしければ、私の EBA、XYZ（cc:）と一緒にスケジュールしてください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)